src/DumpRecorder/DumpRecorder.php

```diff
@@ -18,9 +18,15 @@ class DumpRecorder
 
     protected static $registeredHandler = false;
 
+    protected static $runningLaravel9 = null;
+
     public function __construct(Container $app)
     {
         $this->app = $app;
+
+        if (static::$runningLaravel9 === null) {
+            static::$runningLaravel9 = version_compare(app()->version(), '9.0.0') >= 0;
```

Comment:
If you use `>=` as the third argument, then `version_compare` will return a boolean (which is nicer than comparing manually)

Reply:
I was in such a hurry to wrap this up I missed that in the docs. Thank you :)